### ISSUE TITLE
[Bug]: Potential Infinite IFrame Creation When 'checkSessionIframe' Is undefined

### ISSUE THREAD

**Issue by arlllk** [damienbod/angular-auth-oidc-client#1975]
### Version

17.1.0+

### Please provide the exception or error you saw

```true
When 'checkSessionIframe' is not present in the OpenID configuration, the library appears to create a recursive iframe structure(myiFrameForCheckSession). This results in the application repeatedly initializing itself within nested iframes.
```

### Steps to reproduce the behavior

```true
- Configure an OpenID Connect provider without specifying 'checkSessionIframe' in the '.well-known/openid-configuration'.
 - Start the app
 - Observe the continuous creation of nested iframes
```

### A clear and concise description of what you expected to happen.

```true
The library should handle cases where 'checkSessionIframe' is not defined gracefully, without creating an infinite loop of iframes.
```

### Additional context

This issue appears to have been introduced with commit 367448f1b3c9d33206b01a4808ab76a2d6c3017f [here](https://github.com/damienbod/angular-auth-oidc-client/commit/367448f1b3c9d33206b01a4808ab76a2d6c3017f#diff-dec315070d26d82195b08309ea5550cfd8d36e230b58ab9ddb59a401fe9ff3a4L123), The change seems to have moved an 'if' expression that previously checked if  'checkSessionIframe' was undefined or null before setting the iframe location.

**Comment by ZeroZeroOne-dev** [comment]
Bump. we're currently running into this issue.

**Comment by FabianGosebrink** [comment]
Yep, nice catch. The if is truthy, but it is still running further also with a falsy `checkSessionIframe`. Will fix that after the weekend.

**Comment by FabianGosebrink** [comment]
@arlllk Added an early return to prevent this case you mentioned. Does that fit your needs?

**Comment by arlllk** [comment]
Yes, that PR fixes the issue

**Comment by ZeroZeroOne-dev** [comment]
Will this fix be released soon?

**Comment by jeroenvheel** [comment]
When is this fix coming out?
We had to revert back to v17.0.0
